conformance/src/test/scala/za/co/absa/enceladus/samples/TradeConformance.scala

```diff
@@ -24,21 +24,41 @@ object TradeConformance {
   val countryMT = MappingTable(name = "country", version = 0, hdfsPath = "src/test/testData/country",
     schemaName = "country", schemaVersion = 0)
 
-  val countryRule = MappingConformanceRule(order = 0, mappingTable = "country", controlCheckpoint = true,
+  // Seq(("Kc", "CZK"), ("Rand", "ZAR"), ("SWK","SEK")).toDF
```

Comment:
Do we need these commented out lines?

Reply:
I'd leave them. The mapping tables used are in parquet already. These commands will help if someone someday needs to update these tables.

Reply:
I'd leave a comment to that effect to make it clear why they are still there.

Reply:
Or... I can add mapping table generators in the setup method of the suite.

This will be in line with the code quality discussion today.

Reply:
After thinking about it I've decided to switch to mapping table generators in a different pull request. Possibly in the one fixing the missing errors issue. It will touch the array conformance integration test and it is a good chance to make it a little bit better.
I'm going for the extended comment as you've suggested, for now.